tasks/run_cd4pe_job.rb

```diff
@@ -255,6 +255,28 @@ def get_job_script_and_control_repo
     target_file
   end
 
+  def send_logs_to_cd4pe(output)
+    @logger.log("Sending logs to CD4PE.")
+
+    api_endpoint = File.join(@web_ui_endpoint, @job_owner, 'ajax')
+    payload = {
+      op: 'SavePuppetAgentJobLogs',
+      content: {
+        jobInstanceId: @job_instance_id,
+        output: output
+      },
+    }
+
+    client = CD4PEClient.new(web_ui_endpoint: api_endpoint, job_token: @job_token, ca_cert_file: @ca_cert_file, logger: @logger)
+
+    begin
+      response = client.make_request(:post, api_endpoint, payload.to_json)
```

Comment:
Since we aren't using this response object, let's not assign anything to it. Just call the make_request directly!